test/vbinaryc-microkernel-tester.cc

```diff
@@ -301,6 +301,64 @@ void VBinaryCMicrokernelTester::Test(
   }
 }
 
+void VBinaryCMicrokernelTester::Test(
+    xnn_qs16_vbinary_ukernel_fn vbinary, OpType op_type,
+    xnn_init_qs16_mul_minmax_params_fn init_params,xnn_qs16_requantize_fn requantize) const {
+  xnnpack::ReplicableRandomDevice rng;
+  std::uniform_int_distribution<int32_t> s16dist(
+        0, std::numeric_limits<int16_t>::max());
```

Comment:
The min should be `std::numeric_limits<int16_t>::min()`

Reply:
updated